projects/components/src/table/table.component.scss

```diff
@@ -39,7 +38,7 @@
 .data-row {
   display: flex;
   flex-direction: row;
-  height: 44px;
+  min-height: 44px;
```

Comment:
rows can possibly expand with this change

Reply:
Fixed.

Reply:
hmm.. why do we need both height and min-height?

Reply:
The new flex layout was trying to shrink the row height, so we needed the min-height so flex doesn't override the height. The height was just to address your valid comment about the row expanding.